project/ProjectPlugin.scala

```diff
@@ -110,6 +111,8 @@ object ProjectPlugin extends AutoPlugin {
         "org.typelevel"              %% "cats-effect"            % V.cats,
         "io.chrisdavenport"          %% "log4cats-slf4j"         % V.log4cats,
         "ch.qos.logback"              % "logback-classic"        % V.logbackClassic,
+        "com.nrinaudo"               %% "kantan.csv"             % V.kantan,
+        "com.nrinaudo"               %% "kantan.csv-generic"     % V.kantan,
```

Comment:
Have you tried https://christopherdavenport.github.io/cormorant/ ?

Reply:
Didn't know about kantan :)

Reply:
Looks fine with `kantan` indeed but it would worth it to look also cormorant I think, ignore this comment if you don't